Lib/test/test_platform.py

```diff
@@ -275,6 +275,26 @@ def test_libc_ver(self):
         self.assertEqual(platform.libc_ver(support.TESTFN),
                          ('glibc', '1.23.4'))
 
+    @support.cpython_only
+    def test__comparable_version(self):
+        from platform import _comparable_version as V
+        self.assertEqual(V('1.2.3'), V('1.2.3'))
```

Comment:
If I understood correctly, the function has been added to compare correctly 2.7.9 to 2.7.10 (or a similar issue). Maybe add also a test for that?

Maybe also add a test comparing 2.0 and 2.0.0?

Reply:
> If I understood correctly, the function has been added to compare correctly 2.7.9 to 2.7.10 (or a similar issue). Maybe add also a test for that?

Next line.

Reply:
> Maybe also add a test comparing 2.0 and 2.0.0?

Done. Although this is a case for which the result is different from distutils.version.StrictVersion. I don't think this will add problems in practice (in addition to existing problems of using such guessing function).

Reply:
> Next line.

Oops, I missed it, sorry ;-)